mscore/preferences.cpp

```diff
@@ -1424,6 +1442,13 @@ void PreferenceDialog::apply()
 
 #ifdef USE_PORTMIDI
       prefs.portMidiInput = portMidiInput->currentText();
+      prefs.portMidiOutput = portMidiOutput->currentText();
+      if (static_cast<PortMidiDriver*>(static_cast<Portaudio*>(seq->driver())->mididriver())->isSameCoreMidiIacBus(prefs.portMidiInput, prefs.portMidiOutput)) {
+            QMessageBox msgBox;
+            msgBox.setText("Warning: You used the same CoreMIDI IAC bus for input and output.  This will cause problematic loopback of MuseScore's output midi messages, since they will be sent back in to MuseScore as input, causing confusion.  To avoid this problem, use Mac OS's Audio MIDI Setup to create a dedicated virtual port for MuseScore's Midi output, restart MuseScore, and then select that port as output.  Then other programs may use that deidcated virtual port to receive MuseScore's midi input.");
```

Comment:
If you want to mention a "via", I'd say Spotlight (or maybe Launchpad?) rather than Finder. Puzzling as it is, Finder is the file manager and Spotlight is the search.